api_tests/helpers/request_helper.rb

```diff
@@ -6,5 +6,13 @@ def questions(school_pin, pupil_pin)
     self.class.post(BASE_URL+"/api/questions", :body => {:'schoolPin' => school_pin, :'pupilPin' => pupil_pin})
   end
 
+  def admin_home()
+    self.class.get('https://admin-as-dev-mtc.azurewebsites.net/')
+  end
+
+  def spa_home()
```

Comment:
dont need the ()

Reply:
Also would be good to pull in the URLS from somewhere and not have them hardcoded, I dont think they should be pointing to the dev env anyway.

Reply:
@activemq i agree about storing them in config.  However they do need to point to a containerised instance of the sites that is running on HTTPS.  As some of the headers are only applied by the container software (nginx, for example).  HTTPS redirect is configured specifically for Azure.